documentation/STRUCTURE.md

```diff
@@ -0,0 +1,195 @@
+# Documentation
+
+For the purpose of this proposal document we consider all of the following as documentation:
+
+- documentation in our code within the Fairlearn repository which is currently also hosted on readthedocs
+- Jupyter notebooks
+- project website
+- blog posts
+
+Throughout the document we compare with the documentation of other popular projects such as 
+
+- [scikit-learn](https://scikit-learn.org/)
+- [pandas](https://pandas.pydata.org/)
+
+## Goals
+
+The documentation should be
+
+- discoverable, ideally in a single place as opposed to multiple
+- clear
+- concise when describing individual pieces of functionality
+- detailed when describing entire application scenarios, e.g. in the form of case studies
+- available for the latest version, but if possible also for past versions ([example](https://scikit-learn.org/dev/versions.html))
+- maintainable: it should be simple for maintainers to update/validate
+- without ads (readthedocs always has ads that are shown alongside our documentation)
+
+## Proposal
+
+Like for most projects the website, [fairlearn.org](http://fairlearn.org) will be the central place to look for documentation.
```

Comment:
If this is `fairlearn.org`, will it be a primary landing page for people discovering the project?

If so, it may help to lead with a more user-centric framing that gets at more of "is this for me? how does this help me do things I care about?" and less of "this is what we made and here is how you use it."

Reply:
In https://github.com/kevinrobinson/fairlearn/pull/1/files#r420962578 @romanlutz made a comment about how linking to other projects is an opportunity to express how the team positions this project within the wider ecosystem, just by adding a one line description of those other projects.

Reply:
It may also be useful to mention any existing or intended relationship with interpretml.  I noticed there's a v2 UI over there that's about to start shipping dark (https://github.com/interpretml/interpret-community/pull/223#issuecomment-624873150).  It includes things like comparing model performance across subgroups, which is sometimes relevant for fairness assessment, and abstractly similar to some related issues in this repo (eg, https://github.com/fairlearn/fairlearn/issues/238).

I see there's been some overlap in the teaming, and there are some shared dependencies particularly related to the MS design system.  I'd imagine there are also opportunity costs involved in efforts to couple and generalize across projects.